content/04.discussion.md

```diff
@@ -1,3 +1,25 @@
 ## Discussion
 
-Stub in discussion section   
+Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians across the nation to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
+To our knowledge, this was the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
+We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and create over 40 code- and scientifically-reviewed scalable modules for common downstream cancer genomics analyses which have been used to create new knowledge as well as define new questions.
+We detect and show expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
+Molecular subtyping information was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
+Furthermore, this tumor cohort has yet to be profiled by DNA methylation. 
+Thus, to enable more targeted analyses to interrogate the biology of specific tumor subtypes, we created subtyping modules aligned with WHO molecularly-defined diagnoses.
+By working closely with pathologists and clinicians, we were able to determine a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens and determine mis-diagnosed or mis-identified samples. 
```

Comment:
I've rephrased this based on my reading that 44% of samples were EITHER identified or determined mis-identified. If my reading was incorrect, then a different rephrase would be helpful for clarity.

```suggestion
We worked closely with pathologists and clinicians to either determine a research-grade integrated diagnosis or update incorrectly diagnosed or identified samples for 44% (1260/2840) of biospecimens in the OpenPBTA cohort.
```

Reply:
does this sound better:

"We worked closely with pathologists and clinicians to determine a research-grade integrated diagnosis for 44% (1260/2840) of biospecimens as well as identify incorrectly diagnosed or mis-identified samples in the OpenPBTA cohort. " 

We removed mis-identified samples

Reply:
I like that!